package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "italia-backend",
   "version": "1.6.2",
+  "minAppVersion": "0.0.0",
```

Comment:
also add some docs to the `README` (I like your choice of storing the info in package.json)

Reply:
fixed in 2909928